packages/expo-av/android/src/main/java/expo/modules/av/player/SimpleExoPlayerData.java

```diff
@@ -121,13 +122,19 @@ public void load(final Bundle status, final LoadCompletionListener loadCompletio
 
   @Override
   public synchronized void release() {
+    super.release();
     stopUpdatingProgressIfNecessary();
     if (mSimpleExoPlayer != null) {
       mSimpleExoPlayer.release();
       mSimpleExoPlayer = null;
     }
   }
 
+  @Override
+  protected synchronized double getCurrentPositionSeconds() {
```

Comment:
i'm curious about why this method has `synchronized` lock.

Reply:
Leftover from my previous experiments 😅 
Btw there's a big problem with accessing the `mSimpleExoPlayer` instance across threads. It [should be accessed from one thread only](https://exoplayer.dev/hello-world.html#a-note-on-threading) but in expo-av it is accessed from the main (app) thread, the RN bridge and now from JS thread too, which sometimes leads to spamming logcat with `IllegalStateException: Player is accessed on the wrong thread` messages